Networking/Networking/Remote/StripeRemote.swift

```diff
@@ -46,6 +68,8 @@ private extension StripeRemote {
     enum Path {
         static let connectionTokens = "wc_stripe/connection_tokens"
         static let accounts = "wc_stripe/account/summary"
+        static let orders = "payments/orders"
+        static let captureTerminalPayment = "capture_terminal_payment"
```

Comment:
I confirmed the path https://github.com/woocommerce/woocommerce-gateway-stripe/blob/15f3006c8950409d128f514f6e4effc2668e251f/includes/admin/class-wc-rest-stripe-orders-controller.php#L52